### DISCUSSION TITLE
Should the dotnet/sdk image include libxml2 in order to support dotnet-coverage?

### DISCUSSION THREAD

**Discussion by RobinsonWM** [dotnet/dotnet-docker#4183] [Ideas]
In the `dotnet/sdk` image, if I run `dotnet tool install dotnet-coverage` and then run unit tests with code coverage, the result is an empty coverage report, and this message is logged to the console:
```no-highlight
No code coverage data available. Profiler was not initialized. Verify that glibc (>=2.27), libxml2 and all .NET dependecies are installed.
```

The code coverage tool requires `libxml2`, as documented on this GitHub issue: https://github.com/microsoft/vstest/issues/981#issuecomment-993634968

Since code coverage should be a common thing to do when using the `dotnet/sdk` image, should we consider including `libxml2`?  In my particular case, my CI builds run in a Docker container but not as root, so I have to jump through hoops to install `libxml2`.

Also see https://github.com/microsoft/vstest/issues/3400

**Comment by mthalman** [comment]
That's a great question. I've created an issue for this and we'll discuss it in triage: https://github.com/dotnet/dotnet-docker/issues/4184

**Reply by RobinsonWM** [reply]
Great, thanks!
